website/common/locales/en/front.json

```diff
@@ -285,6 +285,8 @@
   "invalidLoginCredentialsLong": "Uh-oh - your email address / login name or password is incorrect.\n- Make sure they are typed correctly. Your login name and password are case-sensitive.\n- You may have signed up with Facebook or Google-sign-in, not email so double-check by trying them.\n- If you forgot your password, click \"Forgot Password\".",
   "invalidCredentials": "There is no account that uses those credentials.",
   "accountSuspended": "Account has been suspended, please contact <%= communityManagerEmail %> with your User ID \"<%= userId %>\" for assistance.",
+  "accountSuspendedTitle": "Account has been suspended",
+  "accountSuspendedBody": "This account, Profile Name <%= profileName %>, User ID \"<%= userId %>\", has been blocked for breaking the [Community Guidelines](https://habitica.com/static/community-guidelines) or [Terms of Service](https://habitica.com/static/terms). For details or to ask to be unblocked, please email Lemoness at <%= communityManagerEmail %> or ask your parent or guardian to email her. Please copy your Profile Name and User ID into the email.",
```

Comment:
Suggested new text, which removes the profile name and also Lemoness's name so the message still works at times when she's on leave:

```
"This account, User ID \"<%= userId %>\", has been blocked for breaking the [Community Guidelines](https://habitica.com/static/community-guidelines) or [Terms of Service](https://habitica.com/static/terms). For details or to ask to be unblocked, please email our Community Manager at <%= communityManagerEmail %> or ask your parent or guardian to email them. Please copy your User ID into the email and include your Profile Name.", 
```

Also can the new text be copied into the `accountSuspended` key and have the code use that instead of `accountSuspendedBody`? That means the website and API/mobile apps will be using the same message and it also keeps the history of the message in one place in git's log, which will be useful if we ever need to look back through previous versions of the message (complaints from blocked users might occasionally make that useful).